pkg/plugins/manager/loader/loader.go

```diff
@@ -35,34 +37,39 @@ type Loader struct {
 	assetPath          *assetpath.Service
 	log                log.Logger
 	cfg                *config.Cfg
+	features           plugins.FeatureToggles
 
-	errs map[string]*plugins.SignatureError
+	errs       map[string]*plugins.SignatureError
+	grafanaURL string
 }
 
 func ProvideService(cfg *config.Cfg, license plugins.Licensing, authorizer plugins.PluginLoaderAuthorizer,
 	pluginRegistry registry.Service, backendProvider plugins.BackendFactoryProvider, pluginFinder finder.Finder,
-	roleRegistry plugins.RoleRegistry, assetPath *assetpath.Service) *Loader {
+	roleRegistry plugins.RoleRegistry, assetPath *assetpath.Service, features plugins.FeatureToggles) *Loader {
 	return New(cfg, license, authorizer, pluginRegistry, backendProvider, process.NewManager(pluginRegistry),
 		storage.FileSystem(log.NewPrettyLogger("loader.fs"), cfg.PluginsPath), roleRegistry, assetPath,
-		pluginFinder)
+		pluginFinder, features)
 }
 
 func New(cfg *config.Cfg, license plugins.Licensing, authorizer plugins.PluginLoaderAuthorizer,
 	pluginRegistry registry.Service, backendProvider plugins.BackendFactoryProvider,
 	processManager process.Service, pluginStorage storage.Manager, roleRegistry plugins.RoleRegistry,
-	assetPath *assetpath.Service, pluginFinder finder.Finder) *Loader {
+	assetPath *assetpath.Service, pluginFinder finder.Finder, features plugins.FeatureToggles) *Loader {
 	return &Loader{
 		pluginFinder:       pluginFinder,
 		pluginRegistry:     pluginRegistry,
 		pluginInitializer:  initializer.New(cfg, backendProvider, license),
 		signatureValidator: signature.NewValidator(authorizer),
 		processManager:     processManager,
 		pluginStorage:      pluginStorage,
-		errs:               make(map[string]*plugins.SignatureError),
-		log:                log.New("plugin.loader"),
-		roleRegistry:       roleRegistry,
-		cfg:                cfg,
-		assetPath:          assetPath,
+		features:           features,
+
+		errs:         make(map[string]*plugins.SignatureError),
+		log:          log.New("plugin.loader"),
+		roleRegistry: roleRegistry,
+		cfg:          cfg,
+		assetPath:    assetPath,
+		grafanaURL:   "https://grafana.com",
```

Comment:
I think this might be a good opportunity to put a `grafanaComURL` into the plugins cfg struct. We can also re-use that in `pkg/plugins/repo/service.go` as the baseURL.

Reply:
good point